File: Projetos_AfonsoMarcondes/UFCD 10789/Python/Exercicios/Exer4.py
def primo(a):
    while True:
        if a < 2:
            print("**Errado**\nEsse numero não é primo.")
            return
        
        for i in range(2, a):
            if a % i == 0:
                print("**Errado**\nEsse numero não é primo.")
                return
            
        print("**Correto**\nEsse numero é primo.")
        break
            
            
num = int(input("Insira um numero primo: "))

primo(num)

File: Projetos_AfonsoMarcondes/C++/Ex6c - linguagem C/Exer4.c
#include <stdio.h>

int main() {
    int n = 0;
    int ascii = 97;
    
    printf("Insira um Numero: ");
    scanf("%d", &n);
    
    for(int i = 0; i < n; i++){
        printf("\n");
        
        for(int y = 0; y < i + 1; y++){
            printf(" %c", ascii);
        }
        ascii += 1;
    }
    
    return 0;
}
